content/docs/user-guide/project-config/cli-reference.md

```diff
@@ -2,6 +2,1696 @@
+| [`sha256`](#sha256) | Creates a hash value for an O3DE object using the SHA-256 secure hash algorithm. |
+
+
+<!-------------------------------------------------------------->
+
+## `get-global-project`
+
+Gets the global project that is registered to the engine. By default, this is read from `<USER_PATH>/.o3de/Registry/bootstrap.setreg`. You can also specify a file path using `-i`.
+
+### Format
+
+```cmd
+get-global-project [-h] [-i INPUT_PATH]
+```
+
+### Usage
+
+Reads the global project from `<USER_PATH>/.o3de/Registry/bootstrap.setreg`.
+
+```cmd
+o3de.bat get-global-project
+```
+
+Reads the global project from a specified file.
+```cmd
+o3de.bat get-global-project -i <USER_PATH>\.o3de\Registry\my-custom.setreg
+```
+
+### Optional parameters
+
+- **`-h, --help`**
+
+  Shows the help message.
+
+- **`-i INPUT_PATH, --input-path INPUT_PATH`**
+
+  A path to the input file that you want to read the  `/Amazon/AzCore/Bootstrap/project_path` key from. If not supplied, then this command uses the input file `</USER_PATH>/.o3de/Registry/bootstrap.setreg` instead.
+
+
+<!-------------------------------------------------------------->
+
+## `set-global-project`
+
+Sets the specified project as the engine's global project. By default this is set in `C:\Users\USER_PATH\.o3de\Registry\bootstrap.setreg`. You can also specify a file path using `-o`.
+
+If you set a global project, then O3DE tools (such as the Asset Processor and Editor) will use the global project. You can override the global project by specifying a project with the `project-path` parameter when you launch the O3DE tools from the command line.
+
+### Format 
+
+```cmd
+set-global-project [-h] (-pp PROJECT_PATH | -pn PROJECT_NAME)
+                                  [-o OUTPUT_PATH] [-f]
+```
+
+### Usage
+
+Sets the specified project as the global project in `<USER_PATH>/.o3de/Registry/bootstrap.setreg`.
+
+```cmd
+o3de.bat set-global-project -pp PROJECT_PATH
+```
+
+Sets the specified project as the global project in the specified path.
+
+```cmd
+o3de.bat set-global-project -pp PROJECT_PATH -o <USER_PATH>\.o3de\Registry\my-custom.setreg
+```
+
+### Optional parameters
+
+- **`-h, --help`**
+
+  Shows the help message.
+
+- **`-pp PROJECT_PATH, --project-path PROJECT_PATH`**
+
+  The path to the project. The path must contain the project manifest file, `project.json`, unless the `--force` parameter is also used.
+
+- **`-pn PROJECT_NAME, --project-name PROJECT_NAME`**
+
+  The name of the project. You can find the name in the project's `project.json` file.
+
+- **`-o OUTPUT_PATH, --output-path OUTPUT_PATH`**
+
+  The path and filename where the `project_path` key is written. The file should be located in `<USER_PATH>/.o3de/Registry/` and have the file extension `.setreg`. If this parameter is not used, the key is written to `<USER_PATH>/.o3de/Registry/bootstrap.setreg`.
+
+- **`-f, --force`**
+
+  Forces the project path to be set in the supplied setreg file. 
```

Comment:
```suggestion
Forces the project path to be set in the supplied `.setreg` file.
```
Alternative wording (for active voice): "Forcibly sets the project path in the supplied `.setreg` file.